<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SRQGS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_27">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="2,1,2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout"/>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="historyEdit"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="writeEdit"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="openButton">
                <property name="text">
                 <string>Open</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="closeButton">
                <property name="text">
                 <string>Close</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Time:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="timeLCD"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Temperature:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="tempLCD"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Altitude:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="altLCD"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Pressure:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="pressLCD"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Package ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="idLCD"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Frequency:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="freqLCD"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>GPS Lon:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lonLCD"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>GPS Lat:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="latLCD"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCustomPlot" name="axelPlot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCustomPlot" name="gyroPlot" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCustomPlot" name="magPlot" native="true"/>
          </item>
          <item>
           <widget class="QCustomPlot" name="pressPlot" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCustomPlot" name="humPlot" native="true"/>
          </item>
          <item>
           <widget class="QCustomPlot" name="tempPlot" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuName">
     <property name="title">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuBaud_rate">
     <property name="title">
      <string>Baud rate</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action115200"/>
     <addaction name="action9600"/>
    </widget>
    <addaction name="menuName"/>
    <addaction name="menuBaud_rate"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuLogging">
    <property name="title">
     <string>Logging</string>
    </property>
    <widget class="QMenu" name="menuAutoparse">
     <property name="title">
      <string>Logging</string>
     </property>
     <addaction name="actionEnabled"/>
     <addaction name="actionDisabled"/>
    </widget>
    <addaction name="actionFilename"/>
    <addaction name="separator"/>
    <addaction name="menuAutoparse"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuLogging"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action115200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>115200</string>
   </property>
  </action>
  <action name="action9600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9600</string>
   </property>
  </action>
  <action name="action_dev_ttyUSB0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>/dev/ttyUSB0</string>
   </property>
  </action>
  <action name="action_dev_ttyACM0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>/dev/ttyACM0</string>
   </property>
  </action>
  <action name="actionFilename">
   <property name="text">
    <string>Enter path</string>
   </property>
  </action>
  <action name="actionEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </action>
  <action name="actionDisabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disabled</string>
   </property>
  </action>
  <action name="actionOpen_charts">
   <property name="text">
    <string>Open charts</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
